benchmark/perf-tests/README.md

```diff
@@ -24,23 +26,26 @@ for previously executed runs of the tests.
 
 The current lists of tests are:
 * [100 RayCluster test](./100-raycluster/)
+* [100 RayJob test](./100-rayjob/)
 
 
 ## Run a performance test with Kind
 
 You can test clusterloader2 configs using Kind.
 
 First create a kind cluster:
-```
+```sh
 kind create cluster --image=kindest/node:v1.27.3
 ```
 
-Install kuberay;
-```
+Install KubeRay;
+```sh
 helm install kuberay-operator kuberay/kuberay-operator --version 1.1.0
 ```
 
 Run a clusterloader2 test:
+```sh
+clusterloader2 --provider kind --kubeconfig ~/.kube/config --testconfig ./100-rayjob/config.yaml
```

Comment:
Because I think most people can't launch 100 RayCluster custom resources on local Kind cluster, I use RayJob instead.

Reply:
makes sense since 100 rayjobs will shutdown after completion so it does't have to run all at once